src/main/java/gov/nasa/jpl/mbee/mdk/model/GenericTable.java

```diff
@@ -107,42 +166,108 @@ public void setHeaders(List<String> h) {
     @Override
     public List<DocumentElement> visit(boolean forViewEditor, String outputDir) {
         List<DocumentElement> res = new ArrayList<DocumentElement>();
-        DiagramTableTool dtt = new DiagramTableTool();
         if (getIgnore()) {
             return res;
         }
+
+        SessionManager.getInstance().createSession(Application.getInstance().getProject(), "Reading Generic Table");
```

Comment:
Is a session really necessary for using this part of the API? Reading model data typically does not require a session.